src/main/scala/intent/sbt/Runner.scala

```diff
@@ -9,9 +9,9 @@ import java.io.{PrintWriter, StringWriter}
 
 class Framework extends SFramework with
   def name(): String = "intent"
-  def fingerprints(): Array[Fingerprint] = Array(IntentFingerprint)
-  def runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): Runner =
-    new SbtRunner(args, remoteArgs, testClassLoader)
+  def fingerprints(): Array[Fingerprint | Null] = Array(IntentFingerprint)
+  def runner(args: Array[String | Null] | Null, remoteArgs: Array[String | Null] | Null, testClassLoader: ClassLoader | Null): Runner =
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.